specs/SystemCursorId

```diff
@@ -0,0 +1,87 @@
+# Background
+
+In visual hosting, the current cursor image cannot be determined easily by the
+hosting application easily. For example, if the cursor over the WebView is also
+over a textbox inside of the WebView, the cursor should usually be IDC_IBEAM.
+And by default, the cursor is IDC_ARROW. Currently, we return the what the
+cursor should be as an HCURSOR object (After calling LoadCursor on those IDC
+values). The hosting application can then use the HCURSOR object to change the
+current cursor to the correct image.
+
+However, Office already has a well established way of using cursors internally
+by using IDC_* values and there is no easy way to convert the HCURSOR object we
+are returning into an IDC value after it's been created.
+
+This new API is to enable hosting applications like Office to get the current
+IDC value of the cursor instead of the HCURSOR as an option.
+
+
+# Description
+
+The `SystemCursorId` property will return the current system cursor ID reproted
+by the underlying rendering engine for WebView2. It is not meant to return a
+value for any custom cursors, such as those defined by CSS.
+
+It can be used at any time but will only change after a `CursorChanged` event.
+
+
+# Examples
+
+```cpp
+CHECK_FAILURE(m_compositionController->add_CursorChanged(
+    Callback<ICoreWebView2ExperimentalCursorChangedEventHandler>(
+        [this](ICoreWebView2ExperimentalCompositionController* sender, IUnknown* args)
+            -> HRESULT {
+            HRESULT hr = S_OK;
+            HCURSOR cursor;
+            UINT32 cursorId;
+            wil::com_ptr<ICoreWebView2ExperimentalCompositionController2> compositionController2 =
+                m_controller.query<ICoreWebView2ExperimentalCompositionController2>();
+            CHECK_FAILURE(compositionController2->get_SystemCursorId(&cursorId));
+            cursor = ::LoadCursor(nullptr, MAKEINTRESOURCE(cursorId));
+            if (cursor == nullptr)
+            {
+                hr = HRESULT_FROM_WIN32(GetLastError());
+            }
+
+            if (SUCCEEDED(hr))
+            {
+                SetClassLongPtr(
+                    m_appWindow->GetMainWindow() /* HWND */, GCLP_HCURSOR, (LONG_PTR)cursor);
+            }
+            return hr;
+        })
+        .Get(),
+    &m_cursorChangedToken));
+
+
+# API Notes
+
+See [API Details](#api-details) section below for API reference.
+
+# API Details
+
+/// This interface is continuation of the 
+/// ICoreWebView2ExperimentalCompositionController interface.
+[uuid(279ae616-b7cb-4946-8da3-dc853645d2ba), object, pointer_default(unique)]
+interface ICoreWebView2ExperimentalCompositionController2 : IUnknown {
```

Comment:
Please update the IDL to reflect what the API will look like in public. Eg

[uuid(**{new guid}**), object, pointer_default(unique)]
**ICoreWebView2CompositionController2** : **ICoreWebView2CompositionController** {